MANIFEST.in

```diff
@@ -10,3 +10,4 @@ recursive-include radiopadre_kernel/js9 *.html
 
 # needs to be in tarball, copied over by setup script
 recursive-include js9-config *.js *.json
+recursive-include icons *.png *.ico
```

Comment:
missing *.svg ?

Reply:
No. I saved it to the repo for safekeeping, but it's not actually used by radiopadre per se for anything, so we don't need to ship it.

Reply:
Alright, I get it.